www/md_base/src/app/home/home.route.coffee

```diff
@@ -17,3 +17,33 @@ class State extends Config
             templateUrl: "views/#{name}.html"
             name: name
             url: '/'
+
+        bbSettingsServiceProvider.addSettingsGroup
+            name: 'home'
+            caption: 'Home'
+            description: 'Settings to customize the display of Home page'
+            items: [
+                type: 'bool'
+                name: 'lock_panels'
+                caption: 'Lock panels'
+                default_value: false
+            ,
+                type: 'hidden'
+                name: 'panels'
+                default_value: [
+                    name: 'overview'
+                    title: 'overview'
+                    collapsed: false
+                    template: 'views/overview_panel.html'
```

Comment:
If you just use a directive, you would not need to encode the template.

Using a directive will allow to define new panels by plugins. (like I did for force fields https://github.com/buildbot/buildbot/blob/master/www/base/src/app/common/directives/forcefields/forcefields.directive.coffee#L7)